docs/config.md

```diff
@@ -29,6 +29,8 @@ for more details.
 | `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS`  | Comma-separated list of traces source and bytecode instrumentations you want to disable.                                                                                                                                                                        |                                |
 | `OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS`  | Comma-separated list of metrics source instrumentations you want to enable. Set `none` to disable all metric instrumentations.                                                                                                                                  | all available instrumentations |
 | `OTEL_DOTNET_AUTO_METRICS_DISABLED_INSTRUMENTATIONS` | Comma-separated list of metrics source instrumentations you want to disable.                                                                                                                                                                                    |                                |
+| `OTEL_DOTNET_AUTO_LOGS_ENABLED_INSTRUMENTATIONS`     | Comma-separated list of logs source instrumentations you want to enable. Set `none` to disable all metric instrumentations.                                                                                                                                     | all available instrumentations |
```

Comment:
```suggestion
| `OTEL_DOTNET_AUTO_LOGS_ENABLED_INSTRUMENTATIONS`     | Comma-separated list of logs source instrumentations you want to enable. Set to `none` to disable all metric instrumentations.                                                                                                                                     | all available instrumentations |
```

Reply:
Fixed in c0871bfa4e1056981df4aa9f13feb2f48cde03a9 together with other places.